corehq/apps/reports/standard/monitoring.py

```diff
@@ -471,7 +471,7 @@ def all_users(self):
         return sorted(users, key=lambda u: u['username_in_report'])
 
     def paginate_list(self, data_list):
-        if self.pagination:
```

Comment:
i'm looking into it; i think it could be a bit tricky because some of the reports assume that `self.pagination` to return a `DatatablesParams`